templates/openharmony/native/engine/build.bat

```diff
@@ -0,0 +1,81 @@
+@echo off 
+:begin
+echo start build
```

Comment:
Is it needed after integrating with editor?

Reply:
it is deleted